tabs/src/components/sample/SampleQueries.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from "react";
+import { Table, TableCell, OpenOutsideIcon, Button} from '@fluentui/react-northstar';
+import { GRAPH_URL, SAMPLE_QUERIES_URL} from "./TabConstants";
+import * as microsoftTeams from "@microsoft/teams-js";
+import PropTypes from 'prop-types';
+
+FetchSamples.propTypes = {
+    query: PropTypes.string,
+    setQuery: PropTypes.func
+};
```

Comment:
if you want to use propTypes here you should also use them in QueryRunner for consistency

Reply:
I think i already did it

Reply:
it already uses propTypes in QueryRunner, not sure if I'm misunderstanding

Reply:
oh sorry i can't read ._.

Reply:
i'm blind, just saw it in queryrunner too